test/lightning/invocation_test.exs

```diff
@@ -368,16 +373,16 @@ defmodule Lightning.InvocationTest do
     end
 
     test "list_work_orders_for_project/1 returns runs ordered by desc finished_at" do
-      fixture = workflow_job_fixture(workflow_name: "chw-help")
-      job_one = fixture.job
-      trigger = fixture.trigger
+      project = insert(:project)
+      workflow = insert(:workflow, name: "chw-help", project: project)
+      job_one = insert(:job, workflow: workflow)
+      trigger = insert(:trigger, workflow: workflow)
+      dataclip = insert(:dataclip)
```

Comment:
Even though these technically satisfy the test (the functions don't care about edges).

There are helpers for creating workflows:

```elixir
job = build(:job)
trigger = build(:trigger)
workflow =
  build(:workflow, project: project)
  |> with_job(job)
  |> with_trigger(trigger)
  |> with_edge({trigger, job})
  |> insert()
```

The job and trigger models (in this example) are not persisted (the workflow insert persists them), and would need to be reloaded to access their assocs. However they _do_ have their `:id` preset, so you may not need to reload them after the workflow insert if a function only relies on the their id.